### PR TITLE
Implemented percept rule and tests for my_stakeholder_id/1

### PR DESCRIPTION
**Pull request by NickTUD**
As a GOAL programmer, I want to implement the newest percepts in the environment so I can work with them.
- Added init percept rule which inserts my_stakeholder_id in our belief base
- Created some simple tests which can be extended if wanted.

### CODE REVIEW

File: Municipality/test.test2g [hunk 1]
```diff
@@ -36,6 +40,9 @@ test tygron with
 		percept(action_logs(ActList)), bel(actionlog(StakeholderID,Description,ActionID,IncList),
 			not(member(actionlog(StakeholderID,Description,ActionID,IncList), ActList)))
 			leadsto not(bel(actionlog(StakeholderID,Description,ActionID,IncList))).
+			
+		%There are some beliefs that should never be deleted. These tests make sure that never happens.
+		never(not(bel(my_stakeholder_id(_))))).
```
Comment (dionytadema) [inline Municipality/test.test2g]: never not is a double negation, better use always

**Comment by dionytadema** [comment]
:+1:  ziet er goed uit nick!

File: Municipality/tygronInit.mod2g [hunk 2]
```diff
@@ -3,8 +3,14 @@ use dummy as beliefs.
...
 module tygronInit {
 		
+		% MY_STAKEHOLDER_ID PERCEPT
+		% Agents don't switch from stakeholder, so this only needs to be inserted once and should never change
+		if percept(my_stakeholder_id(StakeholderID)) then insert(my_stakeholder_id(StakeholderID)).
+		
 		%%% STAKEHOLDER PERCEPT
 		%%% Stakeholder information doens't update during the session.
+		
+		if percept(my_stakeholder_id(StakeholderID)) then insert(my_stakeholder_id(StakeholderID)).
```
Comment (hmyeh) [inline Municipality/tygronInit.mod2g]: This line and line 8 are the same

**Comment by StefanV7** [comment]
Looks good :+1:

**Comment by levilime** [comment]
there are conflicts, please solve them

**Comment by dionytadema** [comment]
Conflicts have been solved for 3 days now, can we please merge?

**Pull request merged**